rules_python/piptool.py

```diff
@@ -48,8 +48,10 @@ def extract_packages(package_names):
     # Add extracted directories to import path ahead of their zip file
     # counterparts.
     sys.path[0:0] = dirs_to_add
-    existing_pythonpath = os.environ['PYTHONPATH'].split(':')
-    os.environ['PYTHONPATH'] = ':'.join(dirs_to_add + existing_pythonpath)
+    existing_pythonpath = os.environ.get('PYTHONPATH')
+    if existing_pythonpath:
+        dirs_to_add.append(existing_pythonpath.split(':'))
```

Comment:
`append` will do the wrong thing here (adds the whole list as a single element of the first list), you want `extend`

Reply:
thanks, good catch.